compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/inner-memo-value-not-promoted-to-outer-scope-dynamic.expect.md

```diff
@@ -24,69 +24,42 @@ function Component(props) {
 ```javascript
 import { c as _c } from "react/compiler-runtime";
 function Component(props) {
-  const $ = _c(15);
+  const $ = _c(5);
   const item = useFragment(FRAGMENT, props.item);
   useFreeze(item);
   let t0;
-  let T0;
-  let t1;
-  let T1;
   if ($[0] !== item) {
     const count = new MaybeMutable(item);
 
-    T1 = View;
-    T0 = View;
-    if ($[5] === Symbol.for("react.memo_cache_sentinel")) {
-      t1 = <span>Text</span>;
-      $[5] = t1;
-    } else {
-      t1 = $[5];
-    }
```

Comment:
all of this reorders out of the scope, with a huge drop in number of memo slots needed